doc/operator/deploy-operator.md

```diff
@@ -0,0 +1,185 @@
+service/observatorium-xyz-thanos-rule                              ClusterIP   None             <none>        10901/TCP,10902/TCP             52s
+service/observatorium-xyz-thanos-store-shard-0                     ClusterIP   None             <none>        10901/TCP,10902/TCP             51s
+
+NAME                                                                       READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/observatorium-xyz-cortex-query-frontend                    1/1     1            1           59s
+deployment.apps/observatorium-xyz-observatorium-api-gateway                1/1     1            1           62s
+deployment.apps/observatorium-xyz-observatorium-api-gateway-thanos-query   1/1     1            1           60s
+deployment.apps/observatorium-xyz-thanos-query                             1/1     1            1           57s
+deployment.apps/observatorium-xyz-thanos-receive-controller                1/1     1            1           56s
+
+NAME                                                                                  DESIRED   CURRENT   READY   AGE
+replicaset.apps/observatorium-xyz-cortex-query-frontend-7bc9b64c65                    1         1         1       59s
+replicaset.apps/observatorium-xyz-observatorium-api-gateway-56c7fbcc89                1         1         1       62s
+replicaset.apps/observatorium-xyz-observatorium-api-gateway-thanos-query-55999ff96c   1         1         1       60s
+replicaset.apps/observatorium-xyz-thanos-query-599859d6cb                             1         1         1       57s
+replicaset.apps/observatorium-xyz-thanos-receive-controller-f89fdf5c7                 1         1         1       56s
+
+NAME                                                        READY   AGE
+statefulset.apps/observatorium-xyz-thanos-compact           1/1     58s
+statefulset.apps/observatorium-xyz-thanos-receive-default   3/3     53s
+statefulset.apps/observatorium-xyz-thanos-rule              2/2     52s
+statefulset.apps/observatorium-xyz-thanos-store-shard-0     1/1     51s
+```
+
+## Test
+
+### Expose observatorium api gateway for external traffic
+* In Kubernetes
+```shell script
+kubectl -n observatorium patch svc observatorium-xyz-observatorium-api-gateway --type='json' -p '[{"op":"replace","path":"/spec/type","value":"NodePort"}]'
+```
+* In OpenShift
+```shell script
+oc -n observatorium expose svc observatorium-xyz-observatorium-api-gateway
+```
+
+### (Option A) Transmit Metrics via Remote Write Client
+```shell script
+kubectl -n default apply -f https://raw.githubusercontent.com/observatorium/configuration/master/tests/manifests/observatorium-up.yaml
+kubectl wait --for=condition=complete --timeout=5m -n default job/observatorium-up
+````
+Result
+```shell script
+job.batch/observatorium-up condition met
+```
+
+### (Option B) Configure Prometheus Remote Write
```

Comment:
not strictly necessary, but would be cool to have an additional one that shows how to configure kube-prometheus with this